programs/ziti-edge-tunnel/ziti-edge-tunnel.c

```diff
@@ -26,15 +26,18 @@
 #include <ziti/ziti_dns.h>
 #include "model/events.h"
 #include "instance.h"
+#include <file-rotator.h>
+#include <time.h>
 
 #if __APPLE__ && __MACH__
 #include "netif_driver/darwin/utun.h"
 #elif __linux__
 #include "netif_driver/linux/tun.h"
 #elif _WIN32
 #include "netif_driver/windows/tun.h"
+#include "windows/windows-service.h"
 #ifndef MAXPATHLEN
-#define MAXPATHLEN _MAX_PATH
+#define MAXPATHLEN MAX_PATH
```

Comment:
this should really be defined in an include file instead of defining it in each source file that needs it.